packages/jasmine/src/jasmine_runner.js

```diff
@@ -112,6 +113,21 @@ function main(args) {
       noSpecsFound = false
     },
   });
+  
+  if (JUnitXmlReporter) {
```

Comment:
feels like we should have a markdown edit in here somewhere to let users know that the jasmine_node_test rule participates in Bazel's standard test case recording mechanism

Reply:
I updated the docs to mention the reporter now so users might think to leverage them